packages/app/components/Queue/Student/StudentQuestionRephraseModal.tsx

```diff
@@ -0,0 +1,36 @@
+import { RephraseQuestionPayload } from "@koh/common";
+import { Button } from "antd";
+import Modal from "antd/lib/modal/Modal";
+import React, { ReactElement } from "react";
+
+type StudentQuestionRephraseModalProps = {
+  courseId: number;
+  payload: RephraseQuestionPayload;
+  handleClose: (number) => void;
+};
+export default function StudentQuestionRephraseModal({
+  courseId,
+  payload,
+  handleClose,
+}: StudentQuestionRephraseModalProps): ReactElement {
+  return (
+    <Modal
+      visible={true}
+      closable={false}
+      footer={[
+        <Button
+          type={"primary"}
+          key={"continue"}
+          href={`/course/${courseId}/queue/${payload.queueId}?edit_question=true`}
```

Comment:
I like this solution, but I'd prefer if it didn't refresh the entire page. Maybe instead of an `href` here, we could wrap the Button with a [Link](https://nextjs.org/docs/api-reference/next/link), see suggestion above